tests/modules/services/gpg-agent/default-homedir.nix

```diff
@@ -5,7 +5,7 @@ with lib;
 {
   config = {
     services.gpg-agent.enable = true;
-    services.gpg-agent.pinentryPackage = null; # Don't build pinentry package.
+    services.gpg-agent.pinentryPackage = pkgs.cowsay;
```

Comment:
In hindsight, pkgs.pinentry-curses or something like that is a better candidate for tests, because cowsay doesn't set the main program so this line produces a warning.

Reply:
You should use a stub package in either case, which _does_ set `meta.mainProgram`.

But for illustration purposes, using an actual pinentry package name would be better.

Reply:
Fixed